tests/Unit/Http/Controllers/RequestAuthorControllerTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Tests\Unit\Http\Controllers;
+
+use Tests\TestCase;
+use App\Http\Controllers\RequestAuthorController;
+use App\Models\RequestWriter;
+use App\Models\User;
+use Illuminate\Http\Request;
+use App\Repositories\RequestWriter\RequestWriterRepository;
+use App\Repositories\RequestWriter\RequestWriterRepositoryInterface;
+use App\Repositories\User\UserRepository;
+use App\Repositories\User\UserRepositoryInterface;
+use ErrorException;
+use Illuminate\Http\RedirectResponse;
+use Mockery as m;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class RequestAuthorControllerTest extends TestCase
+{
+    /**
+     * A basic unit test example.
+     *
+     * @return void
+     */
+    public function setUp() : void
+    {
+        parent::setUp();
+        $this->users = factory(User::class, 5)->make([
+            'role_id' => 1,
+        ]);
+        $this->requestWriter = factory(RequestWriter::class, 5)->make([
+            'user_id' => 1,
+        ]);
+        $this->requestWriterRepo = m::mock(RequestWriterRepositoryInterface::class)->makePartial();
+        $this->userRepo = m::mock(UserRepositoryInterface::class)->makePartial();
+        $this->requestWriterController = new RequestAuthorController($this->requestWriterRepo, $this->userRepo);
+    }
+
+    public function tearDown() : void
+    {
+        unset($this->requestWriterRepo);
+        unset($this->userRepo);
+        unset($this->requestWriterController);
```

Comment:
thay vì unset lần lượt vậy Mockery có hàm close(); để đóng các instance tạo ra của mockery nhé.
```suggestion
        m::close();
```